cloudbuild.yaml

```diff
@@ -41,29 +41,27 @@
 # This is to ensure that changes being proposed hold for every single environment.
 # 
 # To test a build on dev manually:
-# 1. Make sure you are using the correct glcoud project for your dev environment:
-#    gcloud config set project PROJECT_ID
-# 2. Make sure you are on the dev branch:
-#    git checkout dev
-# 3. Build and deploy the project
+# 1. Make sure you are using the correct glcoud project for your dev environment
+#    (replace DEV_PROJECT_ID with your dev gcloud project id):
```

Comment:
Where is this set?

Reply:
I think this might have been confusing but it's not actually an environment variable set anywhere, I'm just using it as a placeholder in the command